web/modules/custom_taxonomy_color/src/Plugin/Field/FieldFormatter/CustomTaxonomyColorDivFormatter.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Drupal\custom_taxonomy_color\Plugin\Field\FieldFormatter;
+
+use Drupal\Component\Render\FormattableMarkup;
+use Drupal\Core\Field\FormatterBase;
+use Drupal\Core\Field\FieldItemListInterface;
+
+/**
+ * Plugin implementation of the 'custom_taxonomy_color' field formatter.
+ *
+ * @FieldFormatter(
+ *   id = "custom_taxonomy_color_div_formatter",
+ *   label = @Translation("Div element with background color"),
+ *   field_types = {
+ *     "custom_taxonomy_color"
+ *   }
+ * )
+ */
+class CustomTaxonomyColorDivFormatter extends FormatterBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function defaultSettings() {
+    return [] + parent::defaultSettings();
```

Comment:
I think that's a useless method, there just call to a parent class method without any additional data. Actually you may remove that method and the formatter should still work as well.